cmd/av/main.go

```diff
@@ -45,35 +46,33 @@ var rootCmd = &cobra.Command{
 			logrus.WithField("av_version", config.Version).Debug("enabled debug logging")
 		}
 
-		var configDirs []string
+		repoConfigDir := ""
 		repo, err := getRepo()
 		// If we weren't able to load the Git repo, that probably just means the
 		// command isn't being run from inside a repo. That's fine, we just
 		// don't need to bother reading repo-local config.
 		if err != nil {
 			logrus.WithError(err).Debug("unable to load Git repo (probably not inside a repo)")
 		} else {
-			gitDir, err := repo.Git("rev-parse", "--git-dir")
+			gitCommonDir, err := repo.Git("rev-parse", "--git-common-dir")
 			if err != nil {
-				logrus.WithError(err).Warning("failed to determine git root directory")
+				logrus.WithError(err).Warning("failed to determine $GIT_COMMON_DIR")
 			} else {
-				configDirs = append(configDirs, gitDir)
+				gitCommonDir, err = filepath.Abs(gitCommonDir)
+				if err != nil {
+					logrus.WithError(err).Warning("failed to determine $GIT_COMMON_DIR")
+				} else {
+					logrus.WithField("git_common_dir", gitCommonDir).Debug("loaded Git repo")
+					repoConfigDir = filepath.Join(gitCommonDir, "av")
```

Comment:
oh so we were not joining `av` before anywhere for repo level config?

I understand that this is a breaking change, but i guess no one would be setting `.git/config.yaml` for av CLI. This is what we have in our public docs:
https://docs.aviator.co/aviator-cli/configuration

Reply:
Yeah. So, I guess nobody could use per-repo config, unless they read the code.